### ISSUE TITLE
checker for parking checklist

### ISSUE THREAD

**Issue by dneise** [fact-project/shifthelper#90]
This http://fact-project.org/Checklist/ should not result in a mail to everybody anymore, but create a flag in a DB or so.

We need a new check, which checks say 30minutes after a scheduled parking (or after end of astro twilight in case shutdown was scheduled for the middle of the night) if the parking checklist was filled, Otherwise call the current shifter (or parker) with possible fallback.

We make sure by this, that at least 30 minutes after sheduled parking or astro twilight somebody realises that the checklist was not filled...

**Comment by dneise** [comment]
So the checklist now writes a database, the project is called [checklist_page](https://github.com/fact-project/checklist_page) and the table layout can be seen here https://github.com/fact-project/checklist_page/blob/master/db.php#L37

```
CREATE TABLE IF NOT EXISTS park_checklist_filled (
                id INT NOT NULL PRIMARY KEY AUTO_INCREMENT,
                username VARCHAR(100),
                created TIMESTAMP DEFAULT NOW()
```

So next is a check, checking from `shutdown_time + 30minutes` until `startup_time` if there was a `created` entry in this `park_checklist_filled` table, which is between `previous_startup_time` and `now`.

**Comment by dneise** [comment]
was solved in a0fac3b7ff407342f1796f19f4deeffde5c0dfb3
